cf-agent/cf-agent.c

```diff
@@ -587,6 +589,10 @@ static GenericAgentConfig *CheckOpts(int argc, char **argv)
                 }
                 config->agent_specific.agent.show_evaluated_variables = xstrdup(optarg);
             }
+            else if (strcmp(OPTIONS[longopt_idx].name, "skip-bootstrap-policy-run") == 0)
```

Comment:
`StringSafeEqual()`

Reply:
Yeah, but I prefer the similarity with the surrounding checks in this case.